docs/website/docs/developers/design-docs/cuda-hal-driver.md

```diff
@@ -1,7 +1,21 @@
-# IREE CUDA HAL Driver
+---
+hide:
+  - tags
+tags:
+  - GPU
+  - CUDA
+---
+
+# CUDA HAL Driver
```

Comment:
nit: most titles are using sentence case: https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings
```suggestion
# CUDA HAL driver
```